packages/console/src/pages/Connectors/index.tsx

```diff
@@ -25,20 +24,24 @@ const Connectors = () => {
   const isSocial = location.pathname === '/connectors/social';
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const [createType, setCreateType] = useState<ConnectorType>();
-  const { data, error, mutate } = useSWR<ConnectorDTO[], RequestError>('/api/connectors');
+  const { data, error, mutate } = useConnectorGroups();
   const isLoading = !data && !error;
 
   const emailConnector = useMemo(
-    () => data?.find((connector) => connector.enabled && connector.type === ConnectorType.Email),
+    () =>
+      data?.find((connector) => connector.enabled && connector.type === ConnectorType.Email)
+        ?.connectors[0],
```

Comment:
So the data is now a list of connector groups? If yes, then we need to update the code
```ts
const emailConnectorGroup = data.find(({ enabled, type }) => enabled && type === ConnectorType.Email);
const [emailConnector] = emailConnectorGroup;
return emailConnector;
```

Reply:
updated